test/request-response-samples-test.js

```diff
@@ -135,8 +151,15 @@ function generateTest(name, methodName, wsdlPath, headerJSON, securityJSON, requ
 
   suite[name] = function(done){
     if(requestXML) requestContext.expectedRequest = requestXML;
-    if(responseXML) requestContext.responseToSend = responseXML;
+    if (responseXML) {
+      if (wsdlOptions.parseReponseAttachments) {//all LF to CRLF
+          responseXML = responseXML.replace(/\r\n/g, "\n");
```

Comment:
why is this needed?

Reply:
you're converting rn, to n, then you're converting n back to rn.  please revert this.

Reply:
Hi!

It is needed, might be implemented in different way, but needed.

The reason is only CRLF allowed in the multipart/related response. And in linux systems (for example travis-ci) git repo pulled with LF only. However if response xml has LF line endings the standart http multipart response parser cannot parse it so tests fails.

That two lines of code does is to replace all "\n" to "\r\n" .
If line 156 would have been left out `responseXML = responseXML.replace(/\r\n/g, "\n");` and only line 157 would be executed than it might occur that some line endings will have "\r\r\n" and that is not correct of course.

So someone with better regexp knowledge might add a solution to replace all LF to CRLF without replacing the LF parts of CRLF's but it is needed.

Reply:
so you're saying there may be a mix of rn initially, so your strategy is to convert those to n then convert all n to rn?

Reply:
Yes.
These lines are because of I noticed that tests fails on linux system, and the reason was that there were \n -s in the file.
So I replaced all to \n to \r\n in a dirty way I mean taking care of not to replace \r\n to \r\r\n...